apps/rocketchat/template-rocketchat-imagestream.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
```

Comment:
Can you move this image stream into the template-rocketchat.yaml file. You can reuse the parameters defined in there. Will keep the RC bits in 1 file too.

Reply:
will rebase and merge the two objects